src/main/java/org/folio/consortia/controller/UserTenantController.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.consortia.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.consortia.domain.dto.UserTenant;
+import org.folio.consortia.domain.dto.UserTenantCollection;
+import org.folio.consortia.rest.resource.UserTenantsApi;
+import org.folio.consortia.service.UserTenantService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/consortia")
+@RequiredArgsConstructor
+public class UserTenantController implements UserTenantsApi {
+
+  @Autowired
+  private final UserTenantService userTenantService;
+
+  @Override
+  public ResponseEntity<UserTenantCollection> getUserTenants(UUID userId, String username, String tenantId,
+                                                             Integer offset, Integer limit) {
+    UserTenantCollection userTenantCollection;
+    if (userId != null) {
```

Comment:
There should be some sanity check here. Which sets of `userId`, `username` and `tenantId` are allowed ? (for instance, can I use userId and username ?) In non-allowed cases a 400 should be returned.

Reply:
It works based on priority, If request cam with userId and username, we just need userId. I think there are not any reason to request all params even if it happened we can get easily with userId. if say we request username and also tenantId in this case it go to findbyUsernameAndTenantId. Please provide more detailed suggestion how it be sanity check. In this case everything seems fine to me

Reply:
I think we can fetch tenantId from X-OKAPI_TENANT to simplify REST contract, API cliens should provide only userId or username as request params

Reply:
Confirmed with @okolawole-ebsco we decided to remain query parameters (and not use header)